[PATCH 5/7] [clang] NNS improvement: getOriginalDecl changes

mizvekov opened the pull request.

mizvekov committed:
[PATCH 5/7] [clang] NNS improvement: getOriginalDecl changes

erichkeane reviewed (approved):
Appreciate you extracting this.  these are pretty trivial too.

mizvekov committed:
[PATCH 6/7] [clang] improve NestedNameSpecifier (#149748)

This contains the NestedNameSpecifier representation changes originally
from: https://github.com/llvm/llvm-project/pull/147835
